sqlglot/parser.py

```diff
@@ -405,6 +405,8 @@ class Parser(metaclass=_Parser):
         TokenType.WINDOW,
     }
 
+    ALIAS_TOKENS = ID_VAR_TOKENS.copy()
```

Comment:
we don't need to copy here right?

Reply:
Hm, we don't need to copy because the base dialect will forever be as lenient in alias tokens as is in identifier tokens - making the reference in this case - and because in whatever sub-dialects do overwrite it, the set difference will create a copy anyways?

Reply:
yea

Reply:
I think that's fine then without the copy, good catch.